src/Runner.php

```diff
@@ -14,22 +14,50 @@ class Runner
 {
     use IO;
 
-    const VERSION = '0.5.4';
+    const VERSION = '0.6.0';
     const ROBOCLASS = 'RoboFile';
     const ROBOFILE = 'RoboFile.php';
-
+    
+    /**
+     * @var CurrentDir
```

Comment:
There certainly is no class named `CurrentDir`, right?
Same for the other doc-block changes...